include/service/media/renderers/daily_motion_media_renderer.js

```diff
@@ -149,7 +149,7 @@ module.exports = function AudioMediaRendererModule(pb) {
      * @return {Boolean} TRUE if URL points to the main domain and media resource, FALSE if not
      */
     DailyMotionMediaRenderer.isLibyanDomain = function(parsedUrl) {
-        if (util.isString(parsedUrl)) {
+        if (util.isString(parsedUrl)) { // Necessary?
```

Comment:
if I recall correctly these were necessary based on some code paths.  Please review the comment and the other for this file.